Sources/Controllers/Panels/OAOptionsPanelBlackViewController.m

```diff
@@ -108,6 +117,9 @@ - (void)updateMenu
 
 - (void) updateLayout
 {
+    if (!_menuButtonsArray) // if class is not initialized
```

Comment:
Do not use indirect sign of class initialized or not. It even not clear what is it - "initialized"
Looks like `self.isViewLoaded` is needed here